src/components/buttons/Button/button.scss

```diff
@@ -0,0 +1,139 @@
+@import "../../../styles/variables";
+
+.btn{
+    border: 0;
+    border-radius: $border_radius_1_6;
+    padding: .4rem 1.6rem;
+    cursor: pointer;
+    font-style: normal;
+    font-weight: 700;
```

Comment:
Please lets use the global variables from the "__variables.scss" for all the things which are already. In case something is missing is better to add it there in case we are going to need it again, instead of having them in many different files. The same applies for 'font_style', 'font-weight', 'font_size' 'line_height' etc..

Reply:
@georgiganchev-7DIGIT Yes, please reuse all variables as @sebastian-7DIGIT suggests.